addons/medical_blood/functions/fnc_createBlood.sqf

```diff
@@ -1,31 +1,31 @@
 #include "script_component.hpp"
 /*
  * Author: Glowbal
- * Spawn a blood drop.
+ * Creates a blood object and handles its cleanup.
  * Available blood drop classes are blooddrop_1 through blooddrop_4.
  *
  * Arguments:
- * 0: classname of blood drop <OBJECT>
+ * 0: Blood Drop Type <STRING>
  * 1: Position <ARRAY>
  *
  * Return Value:
- * Created blood drop <OBJECT>
+ * Blood Drop <OBJECT>
  *
  * Example:
  * ["blooddrop_2", getPos player] call ace_medical_blood_fnc_createBlood
  *
  * Public: No
  */
 
-params ["_type", "_pos"];
-TRACE_2("creating blood",_type,_pos);
+params ["_type", "_position"];
+TRACE_2("Creating blood",_type,_position);
 
 private _model = GVAR(models) getVariable _type;
 
-private _object = createSimpleObject [_model, [0,0,0]];
-_object setDir random 360;
-_object setPos _pos;
+private _bloodDrop = createSimpleObject [_model, [0, 0, 0]];
+_bloodDrop setDir random 360;
+_bloodDrop setPos _position;
```

Comment:
Do we need a seperate setPos? It creates a additonal network message as blooddrops are global.
Speaking of which, couldn't we make blooddrops local only with the new createSimpleObject local syntax?

Reply:
I think we want them to be global, but agree about whether `setPos` is necessary

Reply:
Surprising results (unless I made a mistake)

`createVehicle` @ [0,0,0] - 492 ms
```
private _start = diag_tickTime; 
for "_i" from 0 to 30000 step 3 do { 
    private _pos = (getPos player) vectorAdd [0,_i,0]; 
    private _z = "Land_LampStreet_small_F" createVehicle [0,0,0]; 
    _z setPos _pos; 
}; 
private _end = diag_tickTime; 
x3 = _end - _start;
```

`createVehicle` @ pos - 4798 ms (and non-precise placement)
```
private _start = diag_tickTime; 
for "_i" from 0 to 30000 step 3 do { 
    private _pos = (getPos player) vectorAdd [0,_i,0]; 
    private _z = "Land_LampStreet_small_F" createVehicle _pos; 
}; 
private _end = diag_tickTime; 
x3 = _end - _start;
```

`createSimpleObject` @ [0,0,0] - 263 ms
```
private _start = diag_tickTime; 
for "_i" from 0 to 30000 step 3 do { 
    private _pos = AGLtoASL ((getPos player) vectorAdd [0,_i,0]); 
    private _z = createSimpleObject ["Land_LampStreet_small_F", [0,0,0]]; 
    _z setPosASL _pos;
}; 
private _end = diag_tickTime; 
x3 = _end - _start;
```

`createSimpleObject` @ pos - 2641 ms
```
private _start = diag_tickTime; 
for "_i" from 0 to 30000 step 3 do { 
    private _pos = AGLtoASL ((getPos player) vectorAdd [0,_i,0]); 
    private _z = createSimpleObject ["Land_LampStreet_small_F", _pos]; 
}; 
private _end = diag_tickTime; 
x3 = _end - _start;
```

Reply:
network traffic vs sqf speed
(unless the game can serialize the create and setPos together?)
we can continue to think about this after merge